modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```diff
@@ -550,7 +550,7 @@ object SwaggerUtil {
                   case t"Int"        => Right(p"IntVar(${Pat.Var(paramName)})")
                   case t"Long"       => Right(p"LongVar(${Pat.Var(paramName)})")
                   case t"BigInt"     => Right(p"BigIntVar(${Pat.Var(paramName)})")
-                  case tpe @ Type.Name(_) =>
+                  case tpe =>
```

Comment:
Actually, I'm realizing that this is not going to work -- This code was previously string-interpolating the type name here, so this'll need a little more finesse. Just a minute.